packages/kbn-test/src/functional_test_runner/cli.ts

```diff
@@ -116,6 +129,8 @@ export function runFtrCli() {
         --grep <pattern>   pattern used to select which tests to run
         --invert           invert grep to exclude tests
         --exclude=file     path to a test file that should not be loaded
```

Comment:
Doesn't this PR make this feature obsolete? I feel like we should delete the old file exclusion logic and rename the new flags to `--include=file` `--exclude=file`